### PR TITLE
add instructions on how to do local development for satellite gui

### PR DESCRIPTION
**Pull request by VinozzZ**

### CODE REVIEW

File: code/front-end/Satellite GUI.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Satellite GUI
+
+Satellite GUI is a [vue.js](https://vuejs.org/) + [TypeScript](https://www.typescriptlang.org/) front-end codebase that provides an UI interface for users to interact with the network.
+
```
Comment (JessicaGreben) [inline code/front-end/Satellite GUI.md]: awesome you are writing a doc on this! thank you.

one thing i like to do for docs is to add a `Purpose` section at the top of the doc so people reading it know if they are reading the correct thing

File: code/front-end/Satellite GUI.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+### Running with a back-end server
+
+This way you will be able to test out how the back-end interacts with the front-end
+
+- Make sure you have [storj-sim](https://github.com/storj/storj/wiki/Test-network) set up correctly
+- If you want your local change to be reflected, run `npm run build` from `web/satellite`
+- Run `storj-sim network setup` to set up the test network
+- Run `storj-sim network run` to start the test network
+- Visit `localhost:10002` to access the satellite GUI
```
Comment (JessicaGreben) [inline code/front-end/Satellite GUI.md]: is the GUI always at this port 10002? How can someone find what the port is set to if it isn't here?

Comment (VinozzZ) [reply code/front-end/Satellite GUI.md]: True, I will add the instruction on how to find the correct port

File: code/front-end/Satellite GUI.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+Satellite GUI is a [vue.js](https://vuejs.org/) + [TypeScript](https://www.typescriptlang.org/) front-end codebase that provides an UI interface for users to interact with the network.
+
+## Development Workflow
+
+There are two ways to start local development environment depending on what you need.
+
+### Prerequisites
+
+- Clone the [storj repo](https://github.com/storj/storj)
+- Make sure you have both [node.js](https://nodejs.org/en/) and [npm](https://www.npmjs.com/get-npm) installed
```
Comment (ifraixedes) [inline code/front-end/Satellite GUI.md]: We should have somewhere which version is node is supported.
I would add it in the `package.json` file (https://docs.npmjs.com/files/package.json#engines) and I will inform here where to check which version should be installed.

**Pull request merged**